lib/parser.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+
+const Parser = require('htmlparser2/lib/Parser')
+
+const DIRECTIVES = [
+  { name: '!doctype', start: '<', end: '>' },
+  { name: '?php', start: '<', end: '>' }
+]
+
+const SELF_CLOSING = [
+  'area',
+  'base',
+  'br',
+  'col',
+  'embed',
+  'hr',
+  'img',
+  'input',
+  'keygen',
+  'link',
+  'meta',
+  'param',
+  'source',
+  'track',
+  'wbr',
+  // Custom (PostHTML)
```

Comment:
Relevant (`include` might be needed to be removed for now, since this eventually could break existing code)